core/src/test/scala/unit/kafka/zk/KafkaZkClientTest.scala

```diff
@@ -157,4 +157,27 @@ class KafkaZkClientTest extends ZooKeeperTestHarness {
     assertFalse(statusAndVersion._1)
     assertEquals(-1, statusAndVersion._2)
   }
-}
\ No newline at end of file
+
+  def testSetGetAndDeletePartitionReassignment() {
```

Comment:
Could you add the @Test annotation?

Reply:
Great catch. Interesting that IntelliJ ran the test even though it was missing that annotation.